packages/@react-types/tabs/src/index.d.ts

```diff
@@ -0,0 +1,30 @@
+/*
+* Copyright 2020 Adobe. All rights reserved.
+* This file is licensed to you under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License. You may obtain a copy
+* of the License at http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software distributed under
+* the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+* OF ANY KIND, either express or implied. See the License for the specific language
+* governing permissions and limitations under the License.
+*/
+
+import {CollectionChildren, DOMProps, Orientation, StyleProps} from '@react-types/shared';
+import {Key} from 'react';
+
+export interface TabsProps<T> extends DOMProps, StyleProps {
```

Comment:
Move the type from react-aria here, and extend that in SpectrumTabsProps. The additional properties that are here but not in the aria type should be moved into the spectrum-specific props (e.g. isQuiet, density, overflowMode, isEmphasized, etc.).